frontend/src/components/menu/index.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { ReactNode } from 'react';
+import { ListGroup, Form } from 'react-bootstrap';
+import classNames from 'classnames';
+import ClickAwayListener from 'react-click-away-listener';
+import './menu.scss';
+
+export interface IMenuItemProps {
+  label: string | number;
+  value?: string | number;
+  disabled?: boolean;
+  onClick?: (value?: string | number) => void;
+}
+
+export const MenuItem = (props: IMenuItemProps) => {
+  const onClick = () => {
+    if (props.onClick) {
+      props.onClick(props.value);
+    }
+  };
+
+  return (
+    <ListGroup.Item disabled={props.disabled} className="Menu-item" onClick={onClick}>
+      {props.label}
+    </ListGroup.Item>
+  );
+};
+
+interface IProps {
+  children?: any;
+  label?: ReactNode;
+  options: IMenuItemProps[];
+  width?: string;
+  filter?: boolean;
```

Comment:
What does `filter` mean in this context?  Does it mean add an input to allow filter?

Reply:
It means hide/show filter box

Reply:
I would recommend `showFilter`